website/package.json

```diff
@@ -1,7 +1,9 @@
 {
-  "name": "docusaurus",
-  "version": "0.0.0",
+  "name": "RISC Zero Developer Website",
+  "description": "RISC Zero Documentation, Education Hub, and Resources",
+  "packageManager": "yarn@4.3.1",
```

Comment:
NOTE: you will need to clear node_modules, and run `yarn` that should self update to the LTS version set in the package.json here.